py/torch_tensorrt/dynamo/utils.py

```diff
@@ -204,10 +204,26 @@ def set_log_level(parent_logger: Any, level: Any) -> None:
     Sets the log level to the user provided level.
     This is used to set debug logging at a global level
     at entry points of tracing, dynamo and torch_compile compilation.
+    It also set log level for c++ torch trt logger
     """
     if parent_logger:
         parent_logger.setLevel(level)
 
+    if level == logging.DEBUG:
+        log_level = trt.ILogger.Severity.VERBOSE
+    elif level == logging.INFO:
+        log_level = trt.ILogger.Severity.INFO
+    elif level == logging.WARNING:
+        log_level = trt.ILogger.Severity.WARNING
+    elif level == logging.ERROR:
+        log_level = trt.ILogger.Severity.ERROR
+    elif level == logging.CRITICAL:
+        log_level = trt.ILogger.Severity.INTERNAL_ERROR
+    else:
+        raise AssertionError(f"{level} is not valid log level")
+
+    torch.ops.tensorrt.set_logging_level(int(log_level))
```

Comment:
What happens in the case that it is a python only build? You might want to use enabled features to gate access to this code to only when the C++ runtime is available.

Reply:
You're right, I see the problem. Updated the change.
```
AttributeError: '_OpNamespace' 'tensorrt' object has no attribute 'set_logging_level'
```